### ISSUE TITLE
[Bug] kb.removeMatches() -- this.staementsMatching is not a function

### ISSUE THREAD

**Issue by ckristo** [linkeddata/rdflib.js#116]
````
    this.removeStatements(this.staementsMatching(subject, predicate, object, why))
                               ^

TypeError: this.staementsMatching is not a function
    at $rdf.IndexedFormula.$rdf.IndexedFormula.removeMatches (/Users/christoph/Documents/Ausbildung/TU/033532 Medieninformatik/Bakk-Arbeit/Arbeit/examples/js/rdflib.js-example/node_modules/rdflib/dist/rdflib-node.js:4254:32)
````

There's a typo here that should be fixed, `this.staementsMatching` => `this.statementsMatching`.

BR, Chris

**Comment by dmitrizagidulin** [comment]
Thanks for the catch! :)

**Comment by timbl** [comment]
Just fixed now in    1148240..cf98a8f  dz_convert_to_node_require -> dz_convert_to_node_require
